### PR TITLE
Enable use of .cmd files as fnlfmt command on Windows

### PR DESCRIPTION
**Pull request by nils-hensel**
Spawn command failed on windows using a fnlfmt.cmd file as formatter.
Fixed by spawning child process in a shell environment.

Also updated dependencies to current versions to fix vulnerabilities.

Tested on Windows 11.

### CODE REVIEW

**Comment by nils-hensel** [comment]
Hello, Robin!

I wanted to try out your vsc extension and ran into some problems on Windows.
This should fix it and I hope it does not cause any problems for non-Windows-users!

Best regards,
Nils

File: package-lock.json [hunk 1]
```diff
@@ -0,0 +1,4119 @@
+{
```
Comment (midouest) [inline package-lock.json]: Please remove this file from the PR and only check in the yarn.lock changes. Thanks!

File: package.json [hunk 2]
```diff
@@ -50,19 +50,19 @@
...
-    "mocha": "^8.1.3",
+    "mocha": "^10.7.3",
+    "ts-loader": "^8.0.3",
     "typescript": "^4.0.2",
     "vscode-test": "^1.4.0",
-    "ts-loader": "^8.0.3",
-    "webpack": "^4.44.1",
-    "webpack-cli": "^3.3.12"
+    "webpack": "^5.95.0",
+    "webpack-cli": "^5.1.4"
```
Comment (midouest) [inline package.json:66]: We need to remove the `--info-verbosity verbose` argument passed to webpack in the package.json `watch` script due to the new webpack version.

**Review by midouest — CHANGES_REQUESTED** [review]
Thank you for fixing this and upgrading the dependencies! I am away from my PC at the moment so I can't validate this, but the extension still works on my Mac so I'm not too worried about it. There are just a couple issues that need to be resolved before merging:

- [x] Remove package-lock.json
- [x] Remove the unsupported `--info-verbosity verbose` webpack argument

File: package-lock.json (hunk 1 above)
Comment (nils-hensel) [reply package-lock.json]: Should I add this file to .gitignore?

File: package.json (hunk 2 above)
Comment (nils-hensel) [reply package.json:66]: Done

File: package-lock.json (hunk 1 above)
Comment (nils-hensel) [reply package-lock.json]: Otherwise done

**Review by midouest — APPROVED** [review]
Looks great! Thanks!

**Pull request merged**